apps/app/src/app/shared/feature/filters/id-filter/id-filter.component.ts

```diff
@@ -0,0 +1,79 @@
+import { CommonModule } from '@angular/common';
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnDestroy,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { ClarityModule, ClrDatagridFilterInterface } from '@clr/angular';
+import { TranslateModule } from '@ngx-translate/core';
+import {
+  debounceTime,
+  distinctUntilChanged,
+  Subject,
+  Subscription,
+} from 'rxjs';
+
+@Component({
+  standalone: true,
+  selector: 'app-id-filter',
+  templateUrl: './id-filter.component.html',
+  styleUrls: ['./id-filter.component.scss'],
+  imports: [ClarityModule, TranslateModule, FormsModule, CommonModule],
+})
+export class IdFilterComponent
+  implements ClrDatagridFilterInterface<string>, OnInit, OnDestroy
+{
+  @Output() changes = new EventEmitter<never>();
+
+  @Input() name = '';
+  @Input() inputValue = '';
+
+  inputSubject = new Subject<string>();
+  subscription: Subscription = new Subscription();
```

Comment:
```suggestion
  subscription: Subscription | null = null;
```

ça permet d'éviter d'avoir à instancier un object alors que tu sais très bien qu'il va être remplacer 3 lignes plus tard